code/embeddings.py

```
@@ -0,0 +1,26 @@
+
+# Select text fields for embedding
+text_fields = ["jobTitle", "job function", "industries"]
+
+# Load pre-trained SentenceTransformer model
+model = SentenceTransformer('all-MiniLM-L6-v2')
+
+# Generate embeddings
+for field in text_fields:
+    print(f"Generating embeddings for {field}...")
+    merged_df[f"{field}_embedding"] = merged_df[field].apply(
+        lambda x: model.encode(str(x)).tolist()
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)